utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -243,7 +243,7 @@ def compare_baseline(self):
         compgen += " -testcase_base "+self._case.get_value("CASEBASEID")
         rc, out, err = run_cmd(compgen, ok_to_fail=True)
 
-        append_status(out.replace("compare","compare baseline", 1) + "/n",sfile="TestStatus")
+        append_status(out.replace("compare","compare baseline", 1),sfile="TestStatus")
```

Comment:
Any idea why the previous developer thought they needed to add a newline? Does append_status add the newline for you?